configure.ac

```diff
@@ -27,4 +27,34 @@ AC_SUBST(LTLIBOBJS)
 
 AC_CONFIG_FILES(Makefile libudffs/Makefile mkudffs/Makefile cdrwtool/Makefile pktsetup/Makefile udffsck/Makefile wrudf/Makefile doc/Makefile)
 
+AC_ARG_ENABLE(debug,
+AS_HELP_STRING([--enable-debug],
+[enable debugging, default: no]),
+[case "${enableval}" in
+    yes) debug=true ;;
+    no)  debug=false ;;
+    *)   AC_MSG_ERROR([bad value ${enableval} for --enable-debug]) ;;
+esac],
+[debug=false])
+
+AM_CONDITIONAL(DEBUG, test x"$debug" = x"true")
+
+AC_ARG_ENABLE(tests,
+AS_HELP_STRING([--enable-tests],
+[enable tests building, default: no]),
+[case "${enableval}" in
+    yes) tests=true ;;
+    no)  tests=false ;;
+    *)   AC_MSG_ERROR([bad value ${enableval} for --enable-tests]) ;;
+esac],
+[tests=false])
+
+AM_CONDITIONAL(TESTS, test x"$tests" = x"true")
+
+AC_C_BIGENDIAN
+AM_CONDITIONAL(WORDS_LITTLEENDIAN, test "x$ac_cv_c_bigendian" = "xno")
+AM_CONDITIONAL(WORDS_BIGENDIAN, test "x$ac_cv_c_bigendian" = "xyes")
```

Comment:
Probably these changes can simplify endianity tests in include/bswap.h (e.g. __BYTE_ORDER macro)

Reply:
I hope so. Is there some action for me from that?

Reply:
Now I did it in 0adadb61c88add40a42d312c50c8f3218412c28a